int16    TrackedNum
int16	 HeadH
int16    HeadV
int16    ShoulderSpin
int16    ShoulderElbowXLeft
int16    ShoulderElbowYLeft
int16    ElbowSpinLeft
int16    ShoulderElbowWristLeft
int16    WristSpinLeft
int16    ElbowWristHandLeft
int16    ShoulderElbowXRight
int16    ShoulderElbowYRight
int16    ElbowSpinRight
int16    ShoulderElbowWristRight
int16    WristSpinRight
int16    ElbowWristHandRight
int16    HandXLeft
int16    HandYLeft
int16    HandXRight
int16    HandYRight
